src/components/MapView/Analyser/AnalysisTable/index.tsx

```diff
@@ -111,6 +128,12 @@ const styles = () =>
     innerAnalysisButton: {
       backgroundColor: '#3d474a',
     },
+    sortLabel: {
+      color: theme.palette.text.primary,
```

Comment:
nice use of the theme! We need to do this more, as you could notice in places like line 129